andela/travel_tool_front
# #161099030 Fix bugs with refreshing protected routes

#### What does this PR do?
This PR fixes redirect on page refresh for some protected routes

#### Description of Task to be completed?
- fix redirect reload on guest house management page
- fix reload on the super-admin page
- fix bug with side navigation

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout bg-fix-request-page`
- Install dependencies - `yarn install`
- Start the server  - `make start`
- Navigate to `localhost:3000` in a browser
- Change your role to travel admin in the database then navigate to residence => manage and refresh the page
- Change your role to super admin in the database then navigate to the page for adding new roles, refresh the page


#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161099030 ](https://www.pivotaltracker.com/story/show/161099030)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
